docs/en/02_Developer_Guides/01_Templates/How_Tos/02_Pagination.md

```diff
@@ -116,6 +116,8 @@ list.
 | `$Pages` | Total number of pages. |
 | `$PageNum` | Page number, starting at 1 (within `$Pages`) |
 | `$Link` | Links to the current controller URL, setting this page as current via a GET parameter |
+| `$FirstPage` | Returns true if you're currently on the first page |
+| `$LastPage` | Returns true if you're currently on the last page |
```

Comment:
There's a `$NotFirstPage` and `$NotLastPage` variable as well which are weird and I'm disinclined to bring attention to.